src/DotNetWorker.Core/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -117,6 +117,37 @@ internal static IServiceCollection AddDefaultInputConvertersToWorkerOptions(this
             });
         }
 
+        internal static IServiceCollection AddDefaultCapabilitiesToWorkerOptions(this IServiceCollection services)
+        {
+            return services.Configure<WorkerOptions>((workerOptions) =>
+            {
+                workerOptions.AddDefaultCapabilities();
+            });
+        }
+
+        internal static void AddDefaultCapabilities(this WorkerOptions workerOptions)
+        {
+            workerOptions.Capabilities["RpcHttpBodyOnly"] = bool.TrueString;
```

Comment:
Do we need some validation in place to ensure some capabilities have the expected value?  Some of these (e.g., `UseNullableValueDictionaryForHttp`) could potentially cause problems if overridden, right?

Reply:
I changed this so there are some immutable values.